searchobject/src/main/java/org/jalau/at18/searchobject/model/facedetector/FaceDetect.java

```diff
@@ -89,7 +90,9 @@ public static void faceDetection (String file, String type) {
                         new Point(rect.x + rect.width, rect.y + rect.height) , new Scalar(0, 0, 255), 3);
             }
             String finalFile = file.substring(22,file.length());
-            Imgcodecs.imwrite("D:\\workspacejala\\progra102\\JU-OBJECT-RECOGNITION\\searchobject\\src\\main\\resources\\images\\"+ finalFile, src);
+            Imgcodecs.imwrite(currentDir + "\\src\\main\\resources\\images\\" + finalFile, src);
+            System.out.println("la imagen   :  "+ currentDir + "\\src\\main\\resources\\images\\" + finalFile);
```

Comment:
delete the system.out.println or replace it with a logger

Reply:
change the System.out.println with logger